src/main/java/frc/robot/commands/ArmCommand.java

```diff
@@ -0,0 +1,144 @@
+  String level;
+  boolean isTop;
+  double integralMinimum = -0.5;
+  double integralMaximum = 0.5;
+  double z;
+  double nodeX;
+  double armSpeed = 0.2;
+  double defaultAngle1 = Math.PI; // angle in radians of joint 1 in default position (vertical)
+  double defaultAngle2 = Math.PI/36; // angle in radians of joint 2 in default position (angle of 5 degrees)
+  double[] defaultAngles;
+  double targetAngularDistanceBottomArm;
+  double targetAngularDistanceTopArm;
+
+  /** Creates a new ArmExtend. */
+  
+  public ArmCommand(String level, boolean isTop) {
+    this.level = level;
+    this.isTop = isTop;
+    // Use addRequirements() here to declare subsystem dependencies.
+    addRequirements(ArmSubsystem.getInstance());
+  }
+
+
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    if (level=="1") {
+      z = 4;//constants --> first and second z positions - depends on the height of the node we are going for
+      nodeX = 19.382;//the x positiion of the node we are going for
+    } 
+    else if(level=="2") {
+      z = 24;
+      nodeX = 37.132;
+    }
+    else if (level=="3") {
+      z = 35;
+      nodeX = 54.132;
+    }
+    else if (level=="default") {
+      defaultAngles = new double[]{defaultAngle1, defaultAngle2};
+      z = 0;
+      nodeX = 0;
+    }
+  }
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+
+    double x = ArmSubsystem.getInstance().setx(0,nodeX);
+    double[] angles = ArmSubsystem.getInstance().calculateAngle(Config.Arm.L1, Config.Arm.L2, x, z);
+    double angle1 = angles[0];
+    double angle2 = angles[1];
+
+    targetAngularDistanceBottomArm = ArmSubsystem.getInstance().getAngularDistance(angle1, Config.Arm.NEO_GEAR_RATIO);
+    targetAngularDistanceTopArm = ArmSubsystem.getInstance().getAngularDistance(angle2, Config.Arm.NEO_GEAR_RATIO);
+    
+    if (level == "3") {
+      // ArmSubsystem.getInstance().setJoint1(angle1); - only used when two motors are involved (testing only)
+      ArmSubsystem.getInstance().setJoint2(angle2); // if level 3 the angle should be 139 degrees, if level 2 the angle should be 74 degrees
+    }
+    else if (level == "2") {
+      ArmSubsystem.getInstance().setJoint1(angle2); // if level 3 the angle should be 59 degrees, if level 2 angle should be 94 degrees
+      // ArmSubsystem.getInstance().setJoint2(angle2); - only used when two motors are involved (testing only)
+    }
+    else if (level == "1") {
+      ArmSubsystem.getInstance().setJoint1(angle2);
+    }
+    else if (level == "default") {
+      ArmSubsystem.getInstance().setDefault(defaultAngles);
+    }
+    // if (ArmSubsystem.getInstance().m_absoluteBottomArmEncoder.getPosition() < targetAngularDistanceBottomArm) {
+    //   ArmSubsystem.getInstance().setJoint1(targetAngularDistanceBottomArm);
+    // }
+    // else {
+    //   ArmSubsystem.getInstance().m_bottomArm.set(0);
+    // }
```

Comment:
I'm not a big fan of committing commented-out code. Is there a reason to leave this in, or can it be taken out?